ml-algorithms/build.gradle

```diff
@@ -24,9 +24,7 @@ dependencies {
     implementation "org.opensearch.client:opensearch-rest-client:${opensearch_version}"
     testImplementation "org.opensearch.test:framework:${opensearch_version}"
     implementation "org.opensearch:common-utils:${common_utils_version}"
-    implementation ("org.jetbrains.kotlin:kotlin-stdlib:${kotlin_version}") {
-       exclude group: "org.jetbrains", module: "annotations"
```

Comment:
@b4sjoo Why remove this part ?

Reply:
I moved this exclusion declaration to plugin level to exclude this package globally.